test/symbol-test.js

```diff
@@ -112,8 +112,8 @@ tape("symbol.type(symbolPentagon) generates a polygon with the specified size",
 
 tape("symbol.type(symbolPentagon) generates the expected path", function(test) {
   var s = shape.symbol().type(extra.symbolPentagon).size(function(d) { return d; });
-  test.pathEqual(s(0), "M0,0L0,0L0,0L0,0L0,0L0,0Z");
-  test.pathEqual(s(10), "M0,-2.050817L0,-2.050817L1.950443,-0.633737L1.205440,1.659146L-1.205440,1.659146L-1.950443,-0.633737Z");
+  test.pathEqual(s(0), "M0,0L0,0L0,0L0,0L0,0Z");
+  test.pathEqual(s(10), "M0,-2.050817L1.950443,-0.633737L1.205440,1.659146L-1.205440,1.659146L-1.950443,-0.633737Z");
```

Comment:
<img width="444" alt="1__vim" src="https://cloud.githubusercontent.com/assets/529247/25559132/e51c1fe0-2d3d-11e7-93c5-a6d934b251ac.png">
And again.